examples/cfchannel/input_cfchannel.in

```diff
@@ -30,7 +29,9 @@ constf1.kx = 1.0
 constf1.ky = 1.0
 constf1.kt = 1.0
 
+
 ###############################################################################
 # Algorithms
 ###############################################################################
 algo.particle_shape = 2
+algo.space_charge = false
```

Comment:
Since space charge can be turned off at various places in the PIC loop, can you remind me which step is disabled when algo.space_charge = false?  For example - is only the space charge push step disabled, or is the Poisson solve also disabled?

Reply:
Absolutely. Currently, this disables:
- coordinate transform z-t
- current deposition
- Poisson solve
- push
- coordinate transform t-z
https://github.com/ECP-WarpX/impactx/blob/34eb424a882c2db73834cbc77e586a6acdc976b6/src/ImpactX.cpp#L126-L157